generators/app/package-json.js

```diff
@@ -124,18 +124,16 @@ const packageJSON = (current, context) => {
       'ie 11'
     ],
     'scripts': _.assign({}, current.scripts, {
-      'prebuild': 'npm run clean',
-      'build': 'npm-run-all -p build:*',
+      'build-test': "npm-run-all -s clean -p 'build:!(js)' 'build:js -- --environment TEST_BUNDLE_ONLY'",
+      'build-prod': "npm-run-all -s clean -p 'build:!(js)' 'build:js -- --environment NO_TEST_BUNDLE'",
+      'build': 'npm-run-all -s clean -p build:*',
       'build:js': 'rollup -c scripts/rollup.config.js',
-      'clean': 'shx rm -rf ./dist ./test/dist',
-      'postclean': 'shx mkdir -p ./dist ./test/dist',
+      'clean': 'shx rm -rf ./dist ./test/dist && shx mkdir -p ./dist ./test/dist',
```

Comment:
May as well do it in one command

Reply:
Although, I don't think`&&` will work on Windows, will it?